homeassistant/components/alert/__init__.py

```diff
@@ -270,6 +274,13 @@ def hidden(self):
         _LOGGER.debug(msg_payload)
 
         for target in self._notifiers:
+            if self._notifier_data.get(target) is not None:
+                data = {}
+                if self._data is not None:
+                    data.update(self._data)
+                data.update(self._notifier_data.get(target))
```

Comment:
Please use `dict[key]` instead of `digt.get` when we know the key is in the dict.